src/web/client/utilities/commonUtil.ts

```diff
@@ -180,6 +180,10 @@ export function getSanitizedFileName(fileName: string): string {
     return fileName.trim().replace(/[`~!@#$%^&*()_|+=?;:'",<>{}[\]\\/]/g, '');
 }
 
+export function getDuplicateFileName(fileName: string, entityId: string): string {
+    return `${fileName}-${entityId}`;
```

Comment:
Should we just display the first part of the guid instead of entire guid (which will be long)?

Reply:
Can be done